.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  CI:
+    continue-on-error: ${{ matrix.distro == 'ubuntu_devel' || matrix.distro == 'fedora_rawhide' || matrix.continue-on-error == true }}
+    strategy:
+      matrix:
+        distro: [latest, fedora_rawhide, opensuse, ubuntu, ubuntu_devel, ubuntu_rolling, ubuntu_18.04]
+        toolchain: [gnu, clang]
+        cmake_build_type: [Release, Debug]
+        minimal: [false]
+        own_gmx: [false]
+        module_build: [false]
+        coverage: [false]
+        no_regression_testing: [true]
+        include:
+          - distro: latest
+            toolchain: gnu
+            cmake_build_type: Debug
+            coverage: RestGroup
+            no_regression_testing: true
+          - distro: latest
+            toolchain: gnu
+            cmake_build_type: Release
+            module_build: true
+            no_regression_testing: true
+    runs-on: ubuntu-latest
+    container: votca/buildenv:${{ matrix.distro }}
+    steps:
+      - uses: actions/checkout@v2.2.0
+        with:
+          submodules: true
+      - name: Run Votca Setup
+        id: setup
+        uses: votca/actions/setup@master
```

Comment:
@junghans this [votca/actions](https://github.com/votca/actions/blob/master/setup/action.yml) just setup all the CMake variables *inside* the ``votca/buildenv:${{ matrix.distro }}``, am I right?

Reply:
Correct! It also checks out the code of the other modules.

Reply:
Actually, the action is always done inside `votca/buildenv:latest`, which sometimes leads to pull of two different containers.